pkg/scheduler/tests/schedulertestutils.go

```diff
@@ -150,6 +150,26 @@ func waitForRejectedApplications(m *MockRMCallbackHandler, appId string, timeout
     }
 }
 
+// Wait till reaches minimum number of accepted Nodes, return duration
+func timeTillMinNumberOfAcceptedNodes(m *MockRMCallbackHandler, minNodes int) time.Duration {
```

Comment:
We need a proper timeout here, in case something wrong we cannot get expected nodes/allocations in time.
Can we replace this with `utils.WaitFor`?

Reply:
@yangwwei Done!